lib/build-secrets/Dockerfile

```diff
@@ -1,8 +1,3 @@
-#TODO: We'll want this to be packaged into a single image,
-#rather than having to install jq every time for example
-FROM %%ARCH%%/alpine:3.10
-
-RUN apk update && apk add bash jq
-
-COPY secrets.json populate.sh ./
-RUN bash populate.sh
+FROM u32915/%%ARCH%%-balena-multibuild-scripts
```

Comment:
* `u32915` is a personal DockerHub account that I've been using for testing, while the new `balena-io-modules/balena-multibuild-scripts` repo is being setup: https://github.com/balena-io-modules/balena-multibuild-scripts/pull/1
* In the end, depending on how `balena-io-modules/balena-multibuild-scripts` gets setup, the available Docker images may be:
  * `balena/arm-balena-multibuild-scripts`
  * `balena/arm64-balena-multibuild-scripts`
  * `balena/amd64-balena-multibuild-scripts`
  * `balena/386-balena-multibuild-scripts`